cmd/doggo/completions.go

```diff
@@ -77,6 +77,8 @@ _doggo() {
     '--color[Colored output]:setting:(true false)' \
     '--debug[Enable debug logging]' \
     '--time[Shows how long the response took from the server]' \
+    '--from[Query using Globalping API from a specific location]' \
+    '--limit[Limit the number of probes to use from Globalping]' \
```

Comment:
Would it make sense to namespace the arg? Because `--limit` might seem to be confused with, maybe limit the number of lookups or resolvers. 

Can we do `--gp-limit` `--gp-from` ?